bin/main.rb

```diff
@@ -1,109 +1,6 @@
 #!/usr/bin/env ruby
-class Game
-  attr_accessor :tic_tac_toe, :player1, :player2
+require_relative '../lib/game.rb'
 
-  def initialize
-    @tic_tac_toe = Board.new
-    puts %(First player; please type your name!)
-    @player1 = Player.new(gets.chomp, 'X')
-    puts %(Second player; please type your name!)
-    @player2 = Player.new(gets.chomp, 'O')
-  end
-
-  def welcome
-    puts %(Welcome on board #{@player1.name} and #{@player2.name}, let the Tic-Tac-Toe game begins!)
-    puts %(Player 1: #{@player1.name} has token: #{@player1.token})
-    puts %(Player 2: #{@player2.name} has token: #{@player2.token})
-  end
-
-  def play
-    welcome
-    loop do
-      @tic_tac_toe.ask_move(@player1)
-      @tic_tac_toe.mark_move(@player1)
-      @tic_tac_toe.show_board
-      @tic_tac_toe.draw?(player1)
-      @tic_tac_toe.winner?(@player2)
-      @tic_tac_toe.ask_move(@player2)
-      @tic_tac_toe.mark_move(@player2)
-      @tic_tac_toe.show_board
-      @tic_tac_toe.draw?(player2)
-      @tic_tac_toe.winner?(@player2)
-    end
-  end
-end
-
-class Board
-  attr_accessor :game_board, :slots_taken
-
-  def initialize
-    @slots_taken = []
-    @game_board = [
-      [' ', ' ', ' '],
-      [' ', ' ', ' '],
-      [' ', ' ', ' ']
-    ]
-  end
-
-  def show_board
-    @game_board.each { |itr| puts %( | #{itr[0]} | | #{itr[1]} | | #{itr[2]} |) }
-  end
-
-  def ask_move(player)
-    puts %(#{player.name}'s turn. Use a number between 1 and 9: )
-    player.next_move = gets.chomp.to_i
-    while marked?(player)
-      puts %(Slot already taken! Choose a different number.)
-      player.next_move = gets.chomp.to_i
-    end
-    until player.next_move.between?(1, 9)
-      puts %(Invalid move. Only use numbres bwteen 1 and 9. Try again: )
-      player.next_move = gets.chomp.to_i
-    end
-    @slots_taken << player.next_move
-  end
```

Comment:
- [ ] Do ensure you remove all `puts` statements here. Kindly see my suggestion above. 
#### **Note: You used the puts statements in other several places in this file and in the board.rb file. Kindly remove all instances of it and put them in your main.rb file.**